internal/merkle/crdt/merklecrdt.go

```diff
@@ -34,31 +35,38 @@ type Stores interface {
 // so it can be merged with any given semantics.
 type MerkleCRDT interface {
 	core.ReplicatedData
-	Clock() core.MerkleClock
-	Save(ctx context.Context, data any) (ipld.Node, uint64, error)
+	Clock() MerkleClocker
+	Save(ctx context.Context, data any) (cidlink.Link, []byte, error)
+}
+
+// MerkleClocker is the logical clock implementation that manages writing to and from
```

Comment:
I moved it from `core` to `merkle/crdt` and added the -er for convention since we also have a `MerkleClock` struct type. But since the struct lives in `merkle/clock`, there wouldn't be any conflict to also naming the interface `MerkleClock`.

Reply:
✅